integration-tests/manager-integration-tests/README.md

```diff
@@ -120,7 +120,7 @@ You can update the parameters needs to configure in `integration-tests/manager-i
 
 In order to be able to run the [performance tests scenarios](src/test/resources/performance-features) locally you will need to have the following installed on your machine:
 
-- [Hyperfoil v0.21](https://hyperfoil.io/) - Benchmark framework for microservices (*)
+- Latest [Hyperfoil](https://hyperfoil.io/) - Benchmark framework for microservices (*)
```

Comment:
Is it stable to always use the latest version. Isn't using the fixed version more consistent and stable?

Reply:
hmm, good point, will mention specific version